tests/Collection/CollectionFunctionalTest.php

```diff
@@ -311,6 +311,71 @@ public function testMapReduce()
         $this->assertNotEmpty($result->getCounts());
     }
 
+    public function testDontInheritWriteConcernInTransaction()
+    {
+        $this->skipIfTransactionsAreNotSupported();
+
+        $this->createCollection();
+
+        $session = $this->manager->startSession();
+        $session->startTransaction([
+            'readConcern' => new \MongoDB\Driver\ReadConcern("snapshot"),
+            'writeConcern' => new \MongoDB\Driver\WriteConcern(\MongoDB\Driver\WriteConcern::MAJORITY)
+        ]);
+
+        $collection = new Collection(
+            $this->manager,
+            $this->getDatabaseName(),
+            $this->getCollectionName(),
+            [ 'writeConcern' => new \MongoDB\Driver\WriteConcern(1) ]
+        );
+
+        (new CommandObserver)->observe(
+            function() use ($collection, $session) {
+                $collection->insertOne(
+                    ['test' => 'foo'],
+                    [
+                        'session' => $session,
+                    ]
```

Comment:
The options seem small enough to rewrite concisely as `['session' => $session]`.